shared/lib_battery_voltage.cpp

```diff
@@ -332,7 +334,12 @@ void voltage_dynamic_t::parameter_compute() {
     _K = ((params->dynamic.Vfull - params->dynamic.Vnom + _A * (std::exp(-_B0 * params->dynamic.Qnom) - 1)) *
           (params->dynamic.Qfull - params->dynamic.Qnom)) / (params->dynamic.Qnom); // [V] - polarization voltage
     _E0 = params->dynamic.Vfull + _K + params->resistance * I - _A;
-
+    if (params->dynamic.Vcut != 0) {
+        double C = (-1 * params->dynamic.Vcut + _E0 - params->resistance * I + _A * (std::exp(-_B0 * params->dynamic.Qfull))) / _K;
+        double x = params->dynamic.Qfull / (C - 1);
+        //params->dynamic.Qfull += x;
+        params->dynamic.Qfull_mod = params->dynamic.Qfull_mod + x;
```

Comment:
What happens if this is called multiple times? This happens in the copy constructor. Will Qfull_mod increase over the simulation?